.gitignore

```diff
@@ -209,7 +209,7 @@ GeneratedArtifacts/
 _Pvt_Extensions/
 ModelManifest.xml
 
-.fake/*
+.fake
```

Comment:
should this change be applied on L214-5 too ?

Reply:
Sure I really only changed it to stop ionide 'suggesting' it should be added